tests/typ/compiler/methods.typ

```diff
@@ -103,33 +103,47 @@
 #test(rgb(1, 2, 3, 4).kind() != luma, true)
 
 ---
-// Test color '.rgba()', '.cmyk()' and '.luma()' without conversions
-#test(rgb(1, 2, 3, 4).to-rgba(), (1, 2, 3, 4))
-#test(rgb(1, 2, 3).to-rgba(), (1, 2, 3, 255))
-#test(cmyk(20%, 20%, 40%, 20%).to-cmyk(), (20%, 20%, 40%, 20%))
-#test(luma(40).to-luma(), 40)
+// Test color '.components()' without conversions
+#test(repr(rgb(1, 2, 3, 4).components()), repr((1, 2, 3, 1.57%)))
```

Comment:
This repr business is due to floating point issues? :/

Reply:
Yes, floats are a bain 😭